### ISSUE TITLE
Grafana install on fresh system failed

### ISSUE THREAD

**Issue by mstormi** [openhab/openhabian#1540]
Installing InfluxDB+Grafana failed.
I installed from the menu on a RPi3 right after installing that from image.

Here's the tail of the install log. Seems Grafana isn't working yet when tested for?
Double the timeout ?

```
+ echo -e '\033[33;01m\nWait for Grafana to start... \033[39;49;00m'

Wait for Grafana to start...
+ curl -4 --retry 6 --retry-connrefused --silent --head http://localhost:3000
+ echo 'FAILED (wait for Grafana to start)'
FAILED (wait for Grafana to start)
+ return 1
+ return 1
+ echo FAILED
FAILED
+ return 1
+ '[' 1 -ne 0 ']'
+ whiptail --msgbox 'There was an error or interruption during the execution of:\n  "20 | Optional Components"\n\nPlease try again. If the error persists, please read /opt/openhabian/docs/openhabian-DEBUG.md or https://github.com/openhab/openhabian/blob/main/docs/openhabian-DEBUG.md how to proceed.' 14 80
```

Some time later, the command that made the scripted install fail proceeds. Maybe we just need a longer timeout ?

[12:02:01] root@smarthouse:/var/lib/openhab/etc# curl -4 --retry 6 --retry-connrefused --silent --head http://localhost:3000; echo $?
HTTP/1.1 302 Found
Cache-Control: no-cache
Content-Type: text/html; charset=utf-8
Expires: -1
Location: /login
Pragma: no-cache
Set-Cookie: redirect_to=%2F; Path=/; HttpOnly; SameSite=Lax
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-Xss-Protection: 1; mode=block
Date: Thu, 17 Jun 2021 10:02:24 GMT

0
```

On a sidenote, if I enter `systemctl status grafana` then hit TAB, it offers TWO completions:
`grafana-server.service` and `grafana.service`. The former is a running service (likely the right one), the latter I don't find anywhere on the system. Is that just an artifact of `systemd/systemctl` ?

**Comment by mstormi** [comment]
Enlarged startup timeout as part of #1541

**Comment by ecdye** [comment]
I'll look at it, but from an initial impression it seems fine.

**Comment by mstormi** [comment]
probably
Fixed-by: #1541
